src/scene/camera.js

```diff
@@ -65,13 +65,19 @@ class Camera {
         this._viewProjMatDirty = true;
 
         this.frustum = new Frustum();
+
+        // Set by XrManager
+        this._xrHorizontalFov = this._horizontalFov;
```

Comment:
Might be neater to group these settings in an object of its own? Then names can mirror the values on the camera, so something like:
```
this.xr = {
    horizonalFov: this._horizontalFov,
    fov: this._fov,
    .... etc
};
```

Reply:
I did wonder that. Would we group the camera component counter parts so that the separation is clearer?

Reply:
`this.xr` is used already, so be careful here. Maybe call it `this.xrProperties`
Also with moving non-xr properties to an object - there could be code directly accessing those in the engine, so maybe leave it as is for now.